.github/workflows/pr.yml

```diff
@@ -94,10 +94,19 @@ jobs:
           run_install: true
       - run: pnpm test
 
+  lint-openapi:
+    name: Lint Open Api specification
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+      - uses: stoplightio/spectral-action@2ad0b9302e32a77c1caccf474a9b2191a8060d83 #v0.8.11
+        with:
+          file_glob: "./packages/api-clients/open-api/!(selfcareV2ClientApi.yml|authorizationManagementApi.yaml|notifierApi.yml)"
```

Comment:
We could also add a way to run this check locally e.g., as a pnpm command in the `api-clients` package, so that one can also check API specs while developing